docs/src/main/asciidoc/security-openid-connect-client.adoc

```diff
@@ -325,6 +315,25 @@ public interface ProtectedResourceService {
 }
 ```
 
+or
+
+```
+import org.eclipse.microprofile.rest.client.annotation.RegisterProvider;
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+import io.quarkus.oidc.token.propagation.AccessTokenRequestFilter;
+
+@RegisterRestClient
+@RegisterProvider(AccessTokenRequestFilter.class)
+@Path("/")
+public interface ProtectedResourceService {
+
+    @GET
+    String getUserName();
+}
+```
+
+Alternatively, `AccessTokenRequestFilter` can be registered automatically with all MP Rest or JAX-RS clients if `quarkus.oidc-client-filter.register-filter=true` property is set.
```

Comment:
@Sgitario good idea, thanks, it should be `quarkus.oidc-token-propagation.register-filter=true` in this case

Reply:
Ops, thanks for spotting! Fixed.

Reply:
@Sgitario Np, the docs build is failing - I've restarted the build, can you check please, if it fails again, may be some extra formatting characters have been introduced, etc